user_sync/app.py

```diff
@@ -52,8 +52,8 @@ def process_args():
                         help='main config filename. (default: "%(default)s")',
                         default=config.DEFAULT_MAIN_CONFIG_FILENAME, metavar='filename', dest='config_filename')
     parser.add_argument('--users', 
-                        help="specify the users to be considered for sync. Legal values are 'all' (the default), 'group name or names' (one or more specified AD groups), 'file f' (a specified input file).", 
-                        nargs="*", metavar=('all|file|group', 'arg1'), dest='users')
+                        help="specify the users to be considered for sync. Legal values are 'all' (the default), 'group name or names' (one or more specified AD groups), 'mapped' (all AD groups listed in configuration file), 'file f' (a specified input file).", 
```

Comment:
do not specify "AD", just say ("all groups listed ...").  There's nothing AD-specific about this.